src/drive/payments/object-storage.ts

```diff
@@ -29,58 +29,56 @@ export class ObjectStorage {
     const query = new URLSearchParams();
     priceId !== undefined && query.set('planId', priceId);
     currency !== undefined && query.set('currency', currency);
-    return this.client.get(`/object-storage-plan-by-id?${query.toString()}`, this.headers());
+    return this.client.get(`/object-storage/price?${query.toString()}`, this.headers());
   }
 
   public createCustomerForObjectStorage({
-    name,
+    customerName,
     email,
     country,
+    postalCode,
     companyVatId,
   }: {
-    name: string;
+    customerName: string;
     email: string;
-    country?: string;
+    country: string;
+    postalCode: string;
     companyVatId?: string;
   }): Promise<{ customerId: string; token: string }> {
-    return this.client.post(
-      '/create-customer-for-object-storage',
-      {
-        name,
-        email,
-        country,
-        companyVatId,
-      },
+    const query = new URLSearchParams();
+    query.set('customerName', customerName);
+    query.set('email', email);
+    query.set('postalCode', postalCode);
+    query.set('country', country);
+    companyVatId !== undefined && query.set('companyVatId', companyVatId);
+
+    return this.client.get(
```

Comment:
What are using a GET method to create a customer?

Reply:
The goal of this endpoint is to get the customer ID, regardless of whether the user already has one or it needs to be created. It’s a retrieval operation from the client’s perspective, which is why we’re using GET.